worker-machines/render.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -eux
+
+source ../config.sh
+source ../lib/common.sh
+
+if [[ -f ./machine-user-data.ign ]]; then
+  cat > ../manifests/managed/machine-user-data-secret.yaml <<EOF
+apiVersion: v1
+kind: Secret
+metadata:
+  name: ${NAMESPACE}-worker-user-data
+  namespace: openshift-machine-api
+data:
+  disableTemplating: $(echo "false" | base64 | tr -d '\n' | tr -d '\r')
+  userData: $(encode ./machine-user-data.ign)
+EOF
+fi
+
+if [[ -f ./machineset.json ]]; then
+  cp ./machineset.json ../manifests/managed/worker-machineset.yaml
+fi
```

Comment:
can you remove temporary artifacts from the directory so we don't have to add things to `.gitignore`?

also, this is writing the secret to `openshift-machine-api` on the management cluster which won't be allowed.  no resources outside the `$NAMESPACE` on the management cluster.

Reply:
Artifacts are created by separate scripts. I wouldn't want to leave the repository in a "dirty" state after running a script.

Because we are using the machine controller on the host cluster, we will need to create artifacts in the openshift-machine-api namespace. My understanding is that the machine controller only syncs resources in that namespace. (This is particular to how we're implementing this on AWS)